client/webserver/locales/en-us.go

```diff
@@ -262,4 +262,9 @@ var EnUS = map[string]string{
 	"No Recent Activity":          "No Recent Activity",
 	"Recent asset_name Activity":  "Recent <span data-asset-name=1></span> Activity",
 	"Other Actions":               "Other Actions",
+	"disable":                     "Disable Wallet",
+	"enable":                      "Enable Wallet",
```

Comment:
`disable_wallet` and `enable_wallet` for the keys pls

Reply:
> disable_wallet and enable_wallet for the keys pls

Alright, but does it mean other  multiple word `key` without `_` in between  will && should be changed to `xx_xx`?

Reply:
Don't need to change them all, but ideally we would have established a convention earlier.  I just prefer a key with no spaced and which is clearly not intended for display.